iceoryx_posh/test/moduletests/test_popo_chunk_sender.cpp

```diff
@@ -183,53 +222,65 @@ TEST_F(ChunkSender_test, freeChunk)
         m_chunkSender.release(chunks[i]);
     }
 
-    EXPECT_THAT(m_memoryManager.getMemPoolInfo(0).m_usedChunks, Eq(0u));
+    EXPECT_THAT(m_memoryManager.getMemPoolInfo(0).m_usedChunks, Eq(0U));
 }
 
 TEST_F(ChunkSender_test, freeInvalidChunk)
```

Comment:
What about a double free test?

Reply:
Shouldn't this be part of the tests for the `MemPool` and `UsedChunkList`? There is no way to check something with the `ChunkSender` except for not segfaulting.